clarifai/client/user.py

```diff
@@ -222,6 +223,46 @@ def runner(self, runner_id: str) -> dict:
 
     return dict(self.auth_helper, check_runner_exists=False, **kwargs)
 
+  def patch_app(self, app_id: str, action: str = 'overwrite', reindex: bool = False,
+                **kwargs) -> App:
+    """Patch an app for the user.
+
+    Args:
+        app_id (str): The app ID for the app to patch.
+        action (str): The action to perform on the app (overwrite/remove).
+        reindex (bool): If set, the app will be automatically reindexed upon change of its base workflow.
```

Comment:
Is reindexing of app through SDK was approved ? Before there was an internal discussion to not to include re-indexing of app through SDK since it costs additionally and user might not be aware of ?

Reply:
Ok, but when we change the base workflow shouldn't we reindex the app data to improve the vector search

Reply:
Yes, We should not let reindex through SDK yet, cc: @sriram-clarifai

Reply:
Done